stackanetes/parameters.yaml

```diff
@@ -54,11 +54,18 @@ ceph:
   glance_user: "glance"
   glance_pool: "images"
   glance_keyring: ""
+  rgw_user: "ironic"
+  admin_keyring: ""
```

Comment:
Maybe put these two below `secret_uuid` and comment that it is only needed for Ironic and shouldn't be provided otherwise.